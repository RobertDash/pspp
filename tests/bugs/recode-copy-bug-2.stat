DATA LIST LIST
 /A (A1)
 B (A1).

BEGIN DATA
1     2
2     3
3     4
END DATA.

STRING a1 (A1).
RECODE A ("1"="3") ("3"="1") (ELSE=COPY) INTO a1.
EXECUTE.
LIST.


