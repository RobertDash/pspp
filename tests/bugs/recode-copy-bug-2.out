1.1 DATA LIST.  Reading free-form data from the command file.
+--------+------+
|Variable|Format|
#========#======#
|A       |A1    |
|B       |A1    |
+--------+------+

A B A1
- - --
1 2  3 
2 3  2 
3 4  1 

